app/src/main/java/com/example/masala_food_recipes/Interactor.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.masala_food_recipes
+
+import com.example.masala_food_recipes.data.DataManager
+import com.example.masala_food_recipes.data.entities.Recipe
+
+class Interactor(context: MainActivity) {
```

Comment:
class name should be meaningful and under 5 ingredient and under 20 recipes should be in separate files each of them should be in one class